pkg/runtime/wfengine/workflow.go

```diff
@@ -209,6 +255,36 @@ func (wf *workflowActor) createWorkflowInstance(ctx context.Context, actorID str
 	return wf.saveInternalState(ctx, actorID, state)
 }
 
+// This method cleans up a workflow associated with the given actorID
+func (wf *workflowActor) cleanupOrchestrationStateInternal(ctx context.Context, actorID string, state *workflowState, requireCompleted bool) error {
+	runtimeState := getRuntimeState(actorID, state)
```

Comment:
We've already loaded the runtime state in `createWorkflowInstance`. Should we reuse that and pass it as a parameter rather than getting the runtime state again?